frontend/components/SingleDocument.js

```diff
@@ -26,7 +26,7 @@ const SingleDocument = ({id}) => {
                 : <div>
                     <h1>{docData.title}</h1>
                     <p>
-                        Author: {docData.author}
+                        Author: {docData.author ? docData.author : "Unknown"}
```

Comment:
Seeing this, I'm wondering if we shouldn't require the existence of an `author` on the `Document`. Did you consider that?

Reply:
I don't think we currently make `author` a required field in the `Document` model.